docs/source/what_is_jupyter.md

```diff
@@ -0,0 +1,412 @@
+# What is Jupyter?
+
+"Jupyter" can mean a lot of things, so let's start from the beginning, and
+break it down:
+
+First and foremost, "Project Jupyter" is a large umbrella project that covers
+many different software offerings and tools. That includes "Jupyter Notebook"
+and "JupyterLab", which are both popular notebook-editor programs. **The**
+**Jupyter project, and its subprojects, all center around providing tools**
+**(and standards) for interactive computing with computational notebooks.**
+If you're not familiar with some of those things, you can take a guided tour
+below that explains each one from the start.
+
+The term "Jupyter" is often used as a shorthand to refer to one of those
+products or ideas, which can cause confusion. Let's take a look at each piece,
+and provide some clarity, below.
+
+# What is a "Computational Notebook" anyway?
+
+A famous computer programmer (Donald Knuth) popularized the idea to combine
+explanatory plain english text with computer code, which is commonly called
+"literate programming". **By adopting this practice, computer programs, as**
+**well as other complex information and ideas, could be better explained to**
+**a wide range of people.**
+
+A program written in this way could be printed on paper and shared by hand
+as an actual "notebook", but in modern times, they are shared digitally as
+"notebook files", and can contain additional rich media like images, 3D models
+and interactive figures, along with data and other program outputs.
+
+By opening a notebook with an editor program like JupyterLab, you can also
+run the code inside the notebook. Since a notebook can contain code that does
+virtually anything, you can do nearly anything regular software can do inside
+of a notebook. For example, a notebook file might be used to:
+
+- Read spreadsheets (or create them) to build reports about your household spending
+- Show proof of your college thesis with interactive graphs and source data
+- Process astronomical data from the James Webb Space Telescope (JWST) (CHECK/LINK)
+- Generate an image of a black hole by processing telescope data (CHECK/LINK)
+- Calculate the presence of gravitational waves from observatory data (CHECK/LINK)
+
+Those last three are real world examples that demonstrate the scientific
+community's usage of computational notebooks (CHECK). Because scientists,
+engineers, and other technical people so often need to perform and communicate
+sophisticated calculations to describe their work, these "computational
+notebooks" became a very popular way to share their work and ideas.
```

Comment:
```suggestion
sophisticated calculations to describe their work, these computational
notebooks became a very popular way to share their work and ideas.
```

Not necessary to quote the term; boldface might be useful to highlight it on first use, if you do so consistently